src/PolygonTool.hpp

```diff
@@ -124,23 +130,23 @@ class PolygonTool : public IVistaOpenGLDraw, public cs::core::tools::MultiPointT
   std::vector<glm::dvec3>        mTriangulation;
   glm::dvec3                     mNormal      = glm::dvec3(0.0);
   glm::dvec3                     mMiddlePoint = glm::dvec3(0.0);
-  bool                           mShowMesh    = 0;
+  bool                           mShowMesh    = false;
   size_t                         mIndexCount2 = 0;
 
   // For triangle fineness
-  float    mHeightDiff = 1.002f;
+  float    mHeightDiff = 1.002F;
   uint32_t mMaxAttempt = 10;
   uint32_t mMaxPoints  = 1000;
   uint32_t mSleekness  = 15;
 
   // For volume calculation
-  double     mOffset;
+  double     mOffset{};
   glm::dvec3 mNormal2      = glm::dvec3(0.0);
   glm::dvec3 mMiddlePoint2 = glm::dvec3(0.0);
 
-  static const int         NUM_SAMPLES;
-  static const std::string SHADER_VERT;
-  static const std::string SHADER_FRAG;
+  static const int   NUM_SAMPLES;
+  static const char* SHADER_VERT;
+  static const char* SHADER_FRAG;
```

Comment:
Still class members?

Reply:
I moved all back to being class members in case we introduce unity builds this avoids name collisions.